### ISSUE TITLE
Is pyRevit_5.0.0.24281_admin_signed the latest?   Why is it not listed on thie site?

### ISSUE THREAD

**Issue by Garyebeck** [pyrevitlabs/pyRevit#2462]

**Comment by jmcouffin** [comment]
Make the effort...

**Comment by Garyebeck** [comment]
Make the effort to do what? I’m a user and not a programmer.

Did pyRevit development split and go in two directions with different programming teams? I can share a link to my folder if needed.

Gary Beck, P. E.
3-D Design & Engineering
Eco-Holdings LLC (F-7395)
4010 Bluebonnet Blvd., Suite 114
Houston, TX 77025
Office: 713-377-4209
Cell: 713-540-1950
Texasengineer.com

On Nov 21, 2024, at 2:02 PM, Jean-Marc Couffin ***@***.***> wrote:

﻿

Make the effort...

**Comment by sanzoghenzo** [comment]
> Make the effort to do what?

Search for previous issues, in the [forum](https://discourse.pyrevitlabs.io) and take a look at the main page of the project, https://www.pyrevitlabs.io/
There you will find that the latest stable version is still 4.8.16, and that there's a link to the latest WIP (5.x) version.

This is a bug reporting or feature request system, your question is neither and should be addressed in the discourse forum - but again, a little search should answer your question.

> Did pyRevit development split and go in two directions with different programming teams?

It's hard to split a team of one 😅 the original developer (@eirannejad) doesn't have any time to spend on the project, @jmcouffin picked up the management of the prohect but he is not a c# developer, @dosymep (which has the most c# experience) helped in adding the support for pyrevit 2025, and I'm helping how I can.

**Comment by Garyebeck** [comment]
Simple questions : Is PyRevit v4.8.16.24121 intended for 2024 Revit?  And the latest WIP (5.x) version intended for 2025 Revit?

I uninstalled all versions of PyRevit and went through Regedit to removed references to pyRevit.
I installed pyRevit v4.8.16.24121 as Administrator. Is it only intended for 2024 Revit?  Since error refer to IronPython I also later installed IronPython, but no change.

I will post this in issues but FYI, I'm running 2025 Revit on Windows 10. I'm trying to get any version of PyRevit to properly apply the 'Match' feature.

**Comment by Garyebeck** [comment]
Andrea,

Thanks.  Simple questions : Is PyRevit v4.8.16.24121 intended for 2024 Revit?  And the latest WIP (5.x) version intended for 2025 Revit?

FYI, I uninstalled all versions of PyRevit and went through Regedit to removed references to pyRevit. I installed pyRevit v4.8.16.24121 as Administrator. Is it only intended for 2024 Revit?  Since error refer to IronPython I also later installed IronPython, but no change.

I will post this in issues. I'm running 2025 Revit on Windows 10. I'm trying to get any version of PyRevit to properly apply the 'Match' feature.

Gary
